src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -48,6 +48,48 @@
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   void method(int i) {
```

Comment:
here is class and method, but other example with property don't have it.
I think it can be removed to make examples similar

Reply:
done.